flow/environments/xsede.py

```diff
@@ -206,11 +206,9 @@ def add_args(cls, parser):
                 "RM",
                 "RM-shared",
                 "RM-small",
-                "LM",
+                "EM",
                 "GPU",
                 "GPU-shared",
-                "GPU-small",
-                "GPU-AI",
             ],
             default="RM-shared",
```

Comment:
Should the default partition be `RM` or `RM-shared`? @jennyfothergill @tcmoore3

Reply:
I only have allocations on GPU partitions... 😅

Reply:
@jennyfothergill Good to know, I hadn’t considered that possibility. I suppose I’ll just leave RM-shared as the default since it is less likely that someone would accidentally use a large number of hours on a shared queue. (RM reserves full nodes.)